### DISCUSSION TITLE
Plotting material output against time step

### DISCUSSION THREAD

**Discussion by batodon** [idaholab/moose#22067] [Q&A Modules: Solid mechanics] [answered]
Hello, how do I plot my material output (example, stress_zz) against my time steps? Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

you can use the output parameter to output material properties to exodus for example. Then you can postprocess the exodus to plot it with time 

Guillaume

**Reply by batodon** [reply]
Thank you. I'm actually outputting my material properties to csv. Does the vectorpostprocessor currently support vectors or tensors now? it seems I cannot postprocess my strains or stresses because they are tensor values (see attached). 
![Capture](https://user-images.githubusercontent.com/113298585/189550270-7b74a762-0d54-4d56-8b86-173fa5074697.PNG)

**Reply by GiudGiud** [reply]
no it doesnt.
This would not be very hard to add and would be a welcome contribution, as I wrote on this page 9 months ago :)
